schedule/sles4sap/cloud-components/ipaddr2.yml

```diff
@@ -6,9 +6,23 @@ vars:
     TEST_CONTEXT: 'OpenQA::Test::RunArgs'
 schedule:
     - boot/boot_to_desktop
+    - '{{validate_repos}}'
     - sles4sap/ipaddr2/deploy
     - sles4sap/ipaddr2/configure
+    - '{{patch_system}}'
+    - sles4sap/ipaddr2/cluster_create
     - sles4sap/ipaddr2/sanity_os
     - sles4sap/ipaddr2/sanity_cluster
     - sles4sap/ipaddr2/test_move_resource
     - sles4sap/ipaddr2/destroy
+conditional_schedule:
+    validate_repos:
```

Comment:
We can keep more test module out from the conditional part. Doing so the test we run in regression and the one we run in maintenance can share more similar scheduling.